src/MojiraBot.ts

```diff
@@ -52,6 +52,7 @@ export default class MojiraBot {
 	public static jira = new JiraClient( {
 		host: 'https://bugs.mojang.com',
 		telemetry: false,
+		newErrorHandling: true
```

Comment:
I looked into changing this but feared that it might mess up error reporting.

If possible we should probably do this in a separate PR.